kubernetes/e2e_test/test_extensions.py

```diff
@@ -0,0 +1,103 @@
+# -*- coding: utf-8 -*-
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+"""
+test_client
```

Comment:
Look like the file comment does not match filename and its content. I suggest either update it or delete the comment if you don't think it adds value. Whatever you decide here, we should apply it to the rest of e2e test files.